packages/liferay-npm-scripts/src/scripts/storybook.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+
+const getMergedConfig = require('../utils/getMergedConfig');
+const log = require('../utils/log');
+const spawnSync = require('../utils/spawnSync');
+
+const BABEL_CONFIG = getMergedConfig('babel');
+const STORYBOOK_CONFIG = getMergedConfig('npmscripts').storybook;
+
+const NODE_PATHS = [
+	path.join(__dirname, '../../../../node_modules'),
+	path.join(__dirname, '../../node_modules')
```

Comment:
😱 — let's keep this one a little secret just between us. 😂

Reply:
If I make a function like this...does it make it better or worse? 😅
```
function getNodeModulesPath(packageName) {
	const pathArray = __dirname.split('/');

	return pathArray
		.slice(0, pathArray.indexOf(packageName) + 1)
		.join('/')
		.concat('/node_modules');
}
```

And then using it:
```
const NODE_PATHS = [
	getNodeModulesPath('liferay-npm-scripts'),
	getNodeModulesPath('liferay-npm-tools')
];
```

Reply:
I like the honesty of the original version better. Fewer things to go wrong, too.

Reply:
👍 I felt somewhat similar after comparing them too. I'll keep it as is.